### PR TITLE
Feature/api setup

### PR DESCRIPTION
**Pull request by zd-prog**

### CODE REVIEW

**Review by C0Rs1K — APPROVED** [review]
Great job!!

**Comment by EgorAntonovich** [comment]
[[BE] API setup](https://trello.com/c/ZymKwbzn/48-be-api-setup)

File: Curtains/Curtains.Application/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Curtains.Application
+{
+    public class Class1
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Application/Class1.cs]: What is it "Class1"?

File: Curtains/Curtains.Application_tests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+namespace Curtains.Application_tests
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Application_tests/UnitTest1.cs]: Please change naming to Curtains.Application.Tests
Also move all tests projects to separated folder with naming Tests

File: Curtains/Curtains.Application_tests/Usings.cs [hunk 3]
```diff
@@ -0,0 +1 @@
+global using NUnit.Framework;
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Application_tests/Usings.cs]: What is it the class?

File: Curtains/Curtains.Domain/Class1.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Curtains.Domain
+{
+    public class Class1
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Domain/Class1.cs]: The same

File: Curtains/Curtains.Domain_tests/UnitTest1.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+namespace Curtains.Domain_tests
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Domain_tests/UnitTest1.cs]: The same as Application

File: Curtains/Curtains.Infrastructure/Class1.cs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Curtains.Infrastructure
+{
+    public class Class1
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Infrastructure/Class1.cs]: ??

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

File: Curtains/Curtains.Application_tests/Usings.cs (hunk 3 above)
Comment (zd-prog) [reply Curtains/Curtains.Application_tests/Usings.cs]: This class was created automaticaly.

**Review by EgorAntonovich — APPROVED** [review]
Looks good for me

**Pull request merged**